content/patterns/button/examples/button_idl.html

```diff
@@ -18,52 +18,52 @@
-        <li><a href="https://github.com/w3c/aria-practices/projects/14">Related Issues</a></li>
-        <li><a href="../button-pattern.html">Design Pattern</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/14">関連する Issues</a></li>
+        <li><a href="../button-pattern.html">デザインパターン</a></li>
       </ul>
     </nav>
 
     <main>
-      <h1>Button Examples (IDL Version)</h1>
+      <h1>ボタンの例（IDL バージョン）</h1>
+      <!-- af8f4c1d97ead402022d704754c3d382c4a2cef2 -->
 
       <section>
-        <h2>About This example</h2>
+        <h2>この例について</h2>
         <img alt="" class="example-page-example-icon" src="../../../images/pattern-button.svg">
-        <p>The following examples of the <a href="../button-pattern.html">Button Pattern</a> demonstrate a new JavaScript syntax for coding ARIA attributes.</p>
+        <p><a href="../button-pattern.html">ボタンパターン</a>の以下の例は、ARIA 属性をコーディングするための新しい JavaScript 構文を示しています。</p>
         <p>
-          The JavaScript for the example buttons on this page uses the <a class="specref" href="#idl-interface">IDL Interface defined in ARIA 1.2</a>.
-          The purpose of these examples is to illustrate how to use ARIA Attribute Reflection and provide a test case for browser and assistive technology interoperability.
-          Specifically, the <code>role</code> and <code>ariaPressed</code> attributes are accessed using dot notation instead of <code>setAttribute()</code> and <code>getAttribute()</code>.
-          In all other respects, these examples are identical to the <a href="button.html">Button Examples</a>.
+          このページの例の JavaScript は、<a class="specref" href="#idl-interface">ARIA 1.2 で定義されている IDL インターフェース</a>を使用しています。
+          これらの例の目的は、ARIA 属性リフレクションの使用方法を示し、ブラウザと支援技術の相互運用性のテストケースを提供することです。
+          具体的には、<code>setAttribute()</code> 及び <code>getAttribute()</code> の代わりにドット表記を使用して <code>role</code> 及び <code>ariaPressed</code> 属性にアクセスします。
+          その他の点では、これらの例は <a href="button.html">ボタンの例</a>と同一です。
         </p>
       </section>
 
       <section>
         <div class="example-header">
-          <h2 id="ex_label">Example</h2>
+          <h2 id="ex_label">例</h2>
         </div>
 
-        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="の開始"></div>
 
         <div id="example">
           <div class="advisement">
-            <h3>Important</h3>
+            <h3>重要</h3>
             <p>
-              This example is coded using syntax that was not introduced until version 1.2 of the ARIA specification.
-              When using a
-              <a href="https://github.com/w3c/aria-practices/issues/1692" target="_blank">browser that does not yet provide support for ARIA attribute reflection</a>,
-              the buttons will not be styled correctly.
+              この例は、ARIA 仕様の版 1.2 まで導入されなかった構文を使用してコーディングされています。
```

Comment:
これは仕様のバージョンのことなので、例外的に「バージョン」を使う箇所ですかね
```suggestion
              この例は、ARIA 仕様のバージョン 1.2 まで導入されなかった構文を使用してコーディングされています。
```